include/sdf/Box.hh

```diff
@@ -65,6 +68,12 @@ namespace sdf
     /// \return A reference to a gz::math::Boxd object.
     public: gz::math::Boxd &Shape();
 
+    /// \brief Calculate and return the Mass Matrix values for the Box
+    /// \param[in] density Density of the box in kg/m^3
+    /// \return A std::optional with gz::math::Inertiald object or std::nullopt
+    public: std::optional<gz::math::Inertiald> CalculateInertial(
+      const double _density);
```

Comment:
I have removed the `const` specifier from double types in `CalculateInertial` functions in 0a6a528bfa0a709c917932f996bd2ce24bb62334. However, since we are[ setting the `gz:math::Material`](https://github.com/jasmeet0915/sdformat/blob/9c6f8c5e8e9821622f953e7d6b721e6e57084eef/src/Box.cc#L127C30-L127C30) for the shape using the density value in the `CalculateInertial` function, we might not be able to make it a `const` function.

Reply:
Ah, yeah, this is not ideal since it won't be obvious to the user that this will actually mutate the object. Can you add to the documentation that in the process of calculating the inertial, the material property of the object will be changed?